ticdc/ticdc-open-api-v2.md

```diff
@@ -810,6 +810,95 @@ curl -X GET http://127.0.0.1:8300/api/v2/changefeeds/test1
+
+**Example 2: The synchronization is not completed**
+
+```json
+{
+  "synced": false,
+  "sink_checkpoint_ts": "2023-11-30 15:26:31.519",
+  "puller_resolved_ts": "2023-11-30 15:26:23.525",
+  "last_synced_ts": "2023-11-30 15:24:30.115",
+  "now_ts": "2023-11-30 15:26:31.511",
+  "info": "The data syncing is not finished, please wait"
+}
+```
+
+This example shows the query result of an ongoing replication task. By checking both the `synced` and `info` fields, you can learn that the replication task is not completed yet and further waiting is expected.
+
+**Example 3: The synchronization status needs further check**
+
+```json
+{
+  "synced":false,
+  "sink_checkpoint_ts":"2023-12-13 11:45:13.515",
+  "puller_resolved_ts":"2023-12-13 11:45:13.525",
+  "last_synced_ts":"2023-12-13 11:45:07.575",
+  "now_ts":"2023-12-13 11:50:24.875",
+  "info":"Please check whether PD is online and TiKV Regions are all available. If PD is offline or some TiKV regions are not available, it means that the data syncing process is complete. To check whether TiKV regions are all available, you can view 'TiKV-Details' > 'Resolved-Ts' > 'Max Leader Resolved TS gap' on Grafana. If the gap is large, such as a few minutes, it means that some regions in TiKV are unavailable. Otherwise, if the gap is small and PD is online, it means the data syncing is incomplete, so please wait"
+}
+```
+
+This API enables you to query the synchronization status even when the upstream cluster encounters disasters. In some cases, you might not be able to directly determine whether the current data replication task of TiCDC is completed or not. In this case, you can query this API, and then check both the `info` field in the result and the current status of the upstream cluster to learn the specific status.
+
+In this example, `sink_checkpoint_ts` is behind `now_ts` in time, either because TiCDC is still catching up with data replication, or because the PD or TiKV has failed. If this is due to TiCDC still catching up with data replication, it means that the replication task is not completed yet. If this is due to a PD or TiKV failure, it means that the replication task is completed. Therefore, you need to check the `info` field to assist in determining the cluster status.
+
+**Example 4: Query error**
+
+```json
+{
+  "error_msg": "[CDC:ErrPDEtcdAPIError]etcd api call error: context deadline exceeded",
+  "error_code": "CDC:ErrPDEtcdAPIError"
+}
+```
+
+After PD in the upstream cluster fails for a long period of time, querying this API will return an error similar to the preceeding one, which does not provide any information for further judgment. Because PD failures directly affect TiCDC data replication, when encountering such errors, you can assume that TiCDC has completed the data replication as much as possible, but data loss might still occurs to the downstream cluster due to PD failures.
```

Comment:
```suggestion
After PD in the upstream cluster fails for a long period of time, querying this API will return an error similar to the preceding one, which does not provide any information for further judgment. Because PD failures directly affect TiCDC data replication, when encountering such errors, you can assume that TiCDC has completed the data replication as much as possible, but data loss might still occur to the downstream cluster due to PD failures.
```